<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-04-16T17:11:36 IST" name="Suite" finished-at="2025-04-16T17:15:16 IST" duration-ms="219669">
    <groups>
    </groups>
    <test started-at="2025-04-16T17:11:36 IST" name="Test" finished-at="2025-04-16T17:15:16 IST" duration-ms="219669">
      <class name="org.api.testCases.StartupUpChartTest">
        <test-method is-config="true" signature="init()[pri:0, instance:org.api.testCases.StartupUpChartTest@77c2494c]" started-at="2025-04-16T17:11:36 IST" name="init" finished-at="2025-04-16T17:11:36 IST" duration-ms="19" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method is-config="true" signature="setup()[pri:0, instance:org.api.testCases.StartupUpChartTest@77c2494c]" started-at="2025-04-16T17:11:38 IST" name="setup" finished-at="2025-04-16T17:11:38 IST" duration-ms="32" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testGetSeriesValues(java.util.Map)[pri:1, instance:org.api.testCases.StartupUpChartTest@77c2494c]" started-at="2025-04-16T17:11:38 IST" name="testGetSeriesValues" data-provider="testDataMap" finished-at="2025-04-16T17:13:37 IST" duration-ms="119233" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{ecu_src=COREGI, iteration=2.0, abs_time=1727453735.874417832, msg_type=NM_Can, status=1.0, max_delta_time=0.3, Wakeup=1727453735.864499221}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetSeriesValues -->
      </class> <!-- org.api.testCases.StartupUpChartTest -->
      <class name="org.api.testCases.EventAnalyzerChart">
        <test-method is-config="true" signature="setup()[pri:0, instance:org.api.testCases.EventAnalyzerChart@185a6e9]" started-at="2025-04-16T17:13:37 IST" name="setup" finished-at="2025-04-16T17:13:37 IST" duration-ms="19" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testGetEventValues(java.util.Map)[pri:1, instance:org.api.testCases.EventAnalyzerChart@185a6e9]" started-at="2025-04-16T17:13:37 IST" name="testGetEventValues" data-provider="testDataMapEvent" finished-at="2025-04-16T17:15:16 IST" duration-ms="98075" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{=45.0, id=278.0, bus=PDCAN13, frame_id=0xcd9ad47, frame_name=FRONT_0CD9AD47_FD, signal=FRONT_ECH_DUTYRATIO_ERR, sender=FRONT, pdu=FRONT_0CD9AD47, timestamp=1739870486.004227, value=0.0}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [FRONT_ECH_DUTYRATIO_ERR] but found [FRONT_ACECH2_DUTYRATIO_ERR]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [FRONT_ECH_DUTYRATIO_ERR] but found [FRONT_ACECH2_DUTYRATIO_ERR]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at org.api.testCases.EventAnalyzerChart.testGetEventValues(EventAnalyzerChart.java:57)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetEventValues -->
      </class> <!-- org.api.testCases.EventAnalyzerChart -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
